src/index.js

```diff
@@ -0,0 +1,155 @@
+import 'bootstrap';
+import './style/main.scss';
+
+const { Project } = require('./projectClass');
+const { Todo } = require('./todoClass');
+
+const createProjectBtn = document.getElementById('btnCreateProject');
+const projectsulEl = document.getElementById('projects-ul');
+const todoDisplayEl = document.getElementById('tododisplay');
+const projectIndexTask = document.getElementById('projectIndexTask');
+const btnCreateTask = document.getElementById('btnCreateTask');
+
+let projects = [];
+
+function setDefaults() {
+  const retrieved = JSON.parse(localStorage.getItem('todolist'));
+  if (!retrieved) {
+    const project1 = new Project('Test');
+    const todo1 = new Todo('First ToDo', 'your first description', 12, 1);
+    const todo2 = new Todo('Second ToDo', 'your second description', 12, 2);
+    project1.items.push(todo1);
+    project1.items.push(todo2);
+    projects.push(project1);
+    localStorage.setItem('todolist', JSON.stringify(projects));
+  }
+}
+setDefaults();
+
+
+const deleteToDo = (e) => {
+  let projectIndex = document.getElementById('projectIndexTask').value;
+  projectIndex = +projectIndex;
+  const toDoIndex = e.srcElement.previousSibling.value;
+  const parentli = e.srcElement.parentNode;
+  console.log(projects[projectIndex]);
+  projects[projectIndex].items.splice(toDoIndex, 1);
+  console.log(projects[projectIndex]);
+  // updateLocalStorage();
+  parentli.remove();
+};
+
+const showToDoItemInDom = (todo, index) => {
+  const todoLiEl = document.createElement('li');
+  const title = document.createElement('button');
+  const panel = document.createElement('div');
+  const divPannel = document.createElement('div');
+  // const
+  const description = document.createElement('p');
+  const dueDate = document.createElement('p');
+  const proprity = document.createElement('p');
+  const deleteTaskBtn = document.createElement('button');
+
+  divPannel.classList.add('row');
+  title.classList.add('accordion', 'col-9');
+  deleteTaskBtn.classList.add('col-3');
+  panel.classList.add('panel');
+
+  const input = document.createElement('input');
+  input.setAttribute('type', 'hidden');
+  input.id = 'taskHidden';
+  input.value = index;
+  deleteTaskBtn.classList.add('btn', 'btn-danger', 'btn-small');
+  deleteTaskBtn.textContent = 'Remove';
+
+  title.textContent = todo.title;
+  description.textContent = `${todo.description} is a prority: ${todo.priority} task needs to complete within ${todo.duedate} `;
+  // dueDate.textContent = todo.duedate;
+  // proprity.textContent = todo.priority;
```

Comment:
- [ ] Please remove commented pieces of code.

Reply:
Done.